platform_tools/odc_stac.ipynb

```diff
@@ -0,0 +1,1095 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e884f6e5",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# odc-stac"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ef820e17-7276-4ad5-a72b-d42b17e379c6",
+   "metadata": {},
+   "source": [
+    "## Why use odc-stac?\n",
```

Comment:
It would be great to add a short sentence to answer the question directly before introducing the sandbox. Readers need to know if this article is for them so they can decide to keep reading or not. Perhaps something like: "odc-stac allows you to load datasets from Digital Earth Africa in your own computing environment." 

You can then go on to say "While DE Africa provides a computing environment, it has these drawbacks: ..." Then go on to mention how odc-stac can overcome these drawbacks.